apps/eth-wrapper/src/utils/WETHConstants.ts

```diff
@@ -0,0 +1,282 @@
+export const WETHwithdraw_function = ["function withdraw(uint wad) public"];
+export const WETH_ABI = `
```

Comment:
Or maybe this is the abi interface:

import IERC20 from "@openzeppelin/contracts/build/contracts/IERC20.json";